src/domain/app/App.tsx

```diff
@@ -69,6 +55,27 @@ const ApolloWrapper: React.FC<PropsWithChildren> = ({ children }) => {
 };
 
 const App: React.FC = () => {
+  const matomoTracker = useMemo(
+    () =>
+      new MatomoTracker({
+        urlBase: getValue(import.meta.env.REACT_APP_MATOMO_URL_BASE, ''),
+        siteId: import.meta.env.REACT_APP_MATOMO_SITE_ID,
+        srcUrl: import.meta.env.REACT_APP_MATOMO_SRC_URL,
+        enabled: import.meta.env.REACT_APP_MATOMO_ENABLED === 'true',
+        configurations: {
+          setDocumentTitle: `${document.location}${document.title}`,
+          setCookieDomain: import.meta.env.REACT_APP_MATOMO_COOKIE_DOMAIN,
+          setDomains: import.meta.env.REACT_APP_MATOMO_DOMAINS
```

Comment:
I found that since setCookieDomain and setDomains match the current domain, these can be left empty / left out entirely and it will still work. Not entirely sure if they should be left around still, they might be useful or maybe not. 🤷

Reply:
I removed these also. I think it's a good approach to leave all unnecessary params out.